README.md

```diff
@@ -260,7 +261,8 @@ However, if you encounter any problems, try the following:
     close as possible
   - add some glue code in `dsl.R` to make using this package smoother
   - adding more semantic components
-  - new version release on CRAN
+  - new version release on
+CRAN
```

Comment:
?

Reply:
I don't really know why it happens :/ As this file is autogenerated :/ Anyway fixed that.